tests/api/auth/backend/test_basic_auth.py

```diff
@@ -55,23 +55,10 @@ def test_success(self):
         clear_db_pools()
 
         with self.app.test_client() as test_client:
-            response = test_client.get("/api/v1/pools", headers={"Authorization": token})
+            response = test_client.get("/api/experimental/pools", headers={"Authorization": token})
             assert current_user.email == "test@fab.org"
 
         assert response.status_code == 200
-        assert response.json == {
```

Comment:
We're not testing the API, so we don't care about the response, just the status code.